website/blog/2021-09-01-preparing-your-app-for-iOS-15-and-android-12.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Preparing Your App for iOS 15 and Android 12
+author: Samuel Susla
+authorTitle: Software Engineer on React Native
+authorURL: https://twitter.com/SamuelSusla
+authorImageURL: https://avatars.githubusercontent.com/u/1733610?v=4
+authorTwitter: SamuelSusla
+tags: [announcement]
+date: 2021-09-01
+---
+
+# Preparing Your App for iOS 15 and Android 12
+
+Hello everyone!
+
+With new mobile OS versions releasing late this year, we recommend preparing your React Native apps beforehand to avoid regressions when the releases become generally available.
+
+## iOS 15
+
+The release date of iOS 15 hasn’t been announced yet, but based on previous iOS releases, it will likely be around September 16th. Please also account for App Store review time if any changes are required to prepare your app for iOS 15.
+
+### What to watch out for
+
+<h4>QuickType Bar</h4>
+
+The way to disable _QuickType_ bar in _[TextInput](https://reactnative.dev/docs/textinput)_ has changed. _QuickType_ bar is the bar above keyboard with three suggested words. In case your UI needs to have the bar hidden, setting [autoCorrect](https://reactnative.dev/docs/textinput#autocorrect) to false no longer disables _QuickType_ bar in iOS 15 like earlier versions. In order to hide the _QuickType_ bar, you need to also set [spellCheck](https://reactnative.dev/docs/textinput#spellcheck-ios) to false. This will disable spell check, the red underlines, in your _TextInput_. Disabling QuickType bar with spell check enabled is no longer an option.
```

Comment:
```suggestion
The way to disable _QuickType_ bar in _[TextInput](/docs/textinput)_ has changed. _QuickType_ bar is the bar above keyboard with three suggested words. In case your UI needs to have the bar hidden, setting [autoCorrect](/docs/textinput#autocorrect) to false no longer disables _QuickType_ bar in iOS 15 like earlier versions. In order to hide the _QuickType_ bar, you need to also set [spellCheck](/docs/textinput#spellcheck-ios) to `false`. This will disable spell check, the red underlines, in your _TextInput_. Disabling QuickType bar with spell check enabled is no longer an option.
```
Please use the internal links for the docs pages. This is only an example, more links need to updated. 🙂

Reply:
Thanks, I've updated the links and wrapped recommended values in code block.